interview-questions.md

```diff
@@ -7,48 +7,48 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is an instance variable in Ruby? How is it different from other variables?
 
-  Your answer:
+  Your answer: An instance variable is similar to state object key-value pairs in React. Instance variables that belong to a class in ruby. They are different from other variables in that they cannot be accessed outside of the class, but only through methods in the class.
 
-  Researched answer:
+  Researched answer: Instance variables are private while other variable are public. Instance variables do not need to be declare, like other variables in Ruby
 
 
 
 2. What is a block in Ruby?
 
-  Your answer:
+  Your answer: A block is like a higher order function in JavaScript. It is a fuction tha tis built into Ruby which takes fucntions -- that the user defines -- as parameters. Blocks often have to do with iterables.
```

Comment:
Totes. Here, the reference to Javascript would totally be acceptable, as it's a foundational language and not an extension of a language/library.